src/android/BluetoothLePlugin.java

```diff
@@ -4630,6 +4678,36 @@ public void onMtuChanged(BluetoothDevice device, int mtu) {
     }
 
     public void onNotificationSent(BluetoothDevice device, int status) {
+      Operation operation = peripheralQueue.peek();
+
+      JSONArray args = operation.args;
+      CallbackContext callbackContext = operation.callbackContext;
+
+      peripheralQueueRemove();
+
+      //If no callback, just return
+      if (callbackContext == null) {
+        return;
+      }
+
+      JSONObject obj = getArgsObject(args);
+
+      JSONObject returnObj = new JSONObject();
+
+      addDevice(returnObj, device);
+
+      //If successfully notified, return value
+      if (status == BluetoothGatt.GATT_SUCCESS) {
+        addProperty(returnObj, keyStatus, statusNotified);
+        addProperty(returnObj, keySent, true);
+        callbackContext.success(returnObj);
+      } else {
+        //Else it failed
+        addProperty(returnObj, keyError, errorNotify);
+        callbackContext.error(returnObj);
+      }
+
+      //TODO: Remove code below (kept for backward compatibility)
```

Comment:
So will this cause two callbacks to be fired until this is removed? That may lead to some confusion, but perhaps we can remove the code below in the next major release.

Reply:
Yes, that's the case right now. I also agree it should be removed in the next major release so to not break others' apps. Maybe add a deprecation notice in the readme?

Reply:
Yah a deprecation notice would be good. If you can make that change, I should be able to merge this weekend.

Reply:
Hi, sorry for the late reply. I've updated the readme.